src/Views/Filter/Location.cshtml

```diff
@@ -81,10 +81,14 @@
                 <div class"form-group">
                   <label class="govuk-label" for="query">
                     School, university or other training provider
-                    @if (@Model.Errors.HasError("query")) {
+                    @if (@Model.Errors.HasError("query"))
+                    {
                       <span class="govuk-error-message">@Model.Errors.GetError("query").Message</span>
                     }
                   </label>
+                  <label class="govuk-label" for="query">
```

Comment:
This should be on line 83, not a new element.

Reply:
I tried that first but it wrapped and looked a bit weird